network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html

```diff
@@ -90,6 +90,41 @@ <h2 class="h4-heading">{% trans "Trending projects" %}</h2>
       <a class="cta-link d-block d-md-none" href="https://www.mozillapulse.org/add">{% trans "Add your project" %}</a>
     </div>
   </div>
+</div>
+
+{% image page.partner_background_image fill-1200x600 as partner_img %}
```

Comment:
we'll probably turn this into its own fragment, but that's something we can do as part of the frontend work.

Reply:
I'll make a comment in this PR about that for some higher exposure on the FE ticket.